articles/virtual-machines/linux/create-upload-generic.md

```diff
@@ -19,12 +19,12 @@ The Azure platform SLA applies to virtual machines running the Linux OS only whe
 * [Linux on Azure - Endorsed Distributions](endorsed-distros.md)
 * [Support for Linux images in Microsoft Azure](https://support.microsoft.com/kb/2941892)
 
-All other, non-Azure Marketplace, distributions running on Azure have a number of prerequisites. This article can't be comprehensive, as every distribution is different. Even if you meet all the criteria below, you may need to significantly tweak your Linux system for it to run properly.
+All other, non-Azure Marketplace, distributions running on Azure have a number of prerequisites. This article can not be comprehensive, as every distribution is different. Even if you meet all the criteria below, you may need to significantly tweak your Linux system for it to run properly.
 
 This article focuses on general guidance for running your Linux distribution on Azure.
 
 ## General Linux Installation Notes
-1. The Hyper-V virtual hard disk (VHDX) format isn't supported in Azure, only *fixed VHD*.  You can convert the disk to VHD format using Hyper-V Manager or the [Convert-VHD](/powershell/module/hyper-v/convert-vhd) cmdlet. If you're using VirtualBox, select **Fixed size** rather than the default (dynamically allocated) when creating the disk.
+1. The Hyper-V virtual hard disk (VHDX) format is not supported in Azure, only *fixed VHD*.  You can convert the disk to VHD format using Hyper-V Manager or the [Convert-VHD](/powershell/module/hyper-v/convert-vhd) cmdlet. If you're using VirtualBox, select **Fixed size** rather than the default (dynamically allocated) when creating the disk.

```

Comment:
```suggestion
1. The Hyper-V virtual hard disk (VHDX) format isn't supported in Azure, only *fixed VHD*.  You can convert the disk to VHD format using Hyper-V Manager or the [Convert-VHD](/powershell/module/hyper-v/convert-vhd) cmdlet. If you're using VirtualBox, select **Fixed size** rather than the default (dynamically allocated) when creating the disk.

```